src/server/node.rs

```diff
@@ -36,7 +36,7 @@ const CHECK_CLUSTER_BOOTSTRAPPED_RETRY_SECONDS: u64 = 3;
 pub fn create_raft_storage<S>(
     engine: RaftKv<S>,
     cfg: &StorageConfig,
-    read_pool: ReadPool,
+    read_pools: Vec<FuturePool>,
```

Comment:
read_pool or future_pools?

Reply:
it is a pool for reading, implemented using FuturePool. Both names are fine.